src/torrent.ts

```diff
@@ -112,7 +112,7 @@ export function saveTorrentFile(
 	const { outputDir } = getRuntimeConfig();
 	const buf = meta.encode();
 	const name = stripExtension(meta.name);
-	const filename = `[${tag}][${tracker}]${name}.torrent`;
+	const filename = `[${tag}][${tracker}]${meta.getFileSystemSafeName()}.torrent`;
```

Comment:
looks like we lost the `stripExtension` bit:

```suggestion
	const filename = `[${tag}][${tracker}]${stripExtension(meta.getFileSystemSafeName())}.torrent`;
```

Reply:
Oops, good eyes

Reply:
Ah good catch, yea that would have been lost after implementing it into Metafile.